Mathlib/Algebra/Homology/DerivedCategory/Ext.lean

```diff
@@ -303,8 +303,54 @@ lemma homAddEquiv_apply (α : Ext X Y n) : homAddEquiv α = α.hom := rfl
 
 end
 
+/-- The postcomposition `Ext X Y a →+ Ext X Z b` with `β : Ext Y Z n` when `a + n = b`. -/
+@[simps!]
+noncomputable def postcomp (β : Ext Y Z n) (X : C) {a b : ℕ} (h : a + n = b) :
+    Ext X Y a →+ Ext X Z b :=
+  AddMonoidHom.mk' (fun α => α.comp β h) (by simp)
+
+/-- The precomposition `Ext Y Z a →+ Ext X Z b` with `α : Ext X Y n` when `n + a = b`. -/
+@[simps!]
+noncomputable def precomp (α : Ext X Y n) (Z : C) {a b : ℕ} (h : n + a = b) :
+    Ext Y Z a →+ Ext X Z b :=
+  AddMonoidHom.mk' (fun β => α.comp β h) (by simp)
```

Comment:
I think it would be useful to package them as a bilinear map `Ext Y Z a →+ Ext X Y b →+ Ext X Z c` instead, as we have some API for them (e.g. `AddMonoidHom.flip`)

Reply:
I believe that a definition like `covariantSequence` https://github.com/leanprover-community/mathlib4/pull/14515/files#diff-54c3222a1a7565f433c9e06be02de04bc9070035460c229c3f05ca8a4fee701dR133-R138 is clearer using `postcomp` and the dot notation (and soon, there will be the dual construction with `precomp`), rather than some bilinear map evaluated at an element, and half of the times, it will have to be flipped. The best I could try may be defining the bilinear map, but continue using `postcomp` and `precomp` (which would then be abreviations).

Reply:
I have added a new definition `bilinearComp` and made `post/precomp` abbreviations.

Reply:
Thanks for the suggestion!